algorithms/src/snark/marlin/marlin.rs

```diff
@@ -234,6 +234,8 @@ where
         .map_err(SNARKError::from)
     }
 
+    /// Prove that the verifying key indeed includes a part of the reference string,
+    /// as well as the indexed circuit (i.e. the circuit as a set of linear-sized polynomials).
```

Comment:
I think this could probably use a Rustdoc comment linking to the verifying key struct

```suggestion
    /// Prove that the [VerifyingKey] indeed includes a part of the reference string,
    /// as well as the indexed circuit (i.e. the circuit as a set of linear-sized polynomials).
```